.buildkite/hooks/pre-exit

```diff
@@ -2,14 +2,23 @@
 
 set -euo pipefail
 
-source .buildkite/hooks/scripts/util.sh
+unset_secrets () {
```

Comment:
In the future we could think about complete deleting this hook. All agents (k8s or vms) are ephemeral.

Reply:
I will create a separate PR to delete it.